The other failure is a known flaky;

```
=== RUN   TestDockerSwarmSuite/TestSwarmLockUnlockCluster
    docker_cli_swarm_test.go:1112: [deafd89436346] joining swarm manager [d98c044a4dea8]@0.0.0.0:2477, swarm listen addr 0.0.0.0:2478
    docker_cli_swarm_test.go:1113: [d1f5c6938d03f] joining swarm manager [d98c044a4dea8]@0.0.0.0:2477, swarm listen addr 0.0.0.0:2479
    docker_cli_swarm_test.go:1001: timeout hit after 1s: first check never completed
    daemon_swarm.go:162: assertion failed: error is not nil: error during connect: Get "http://%2Ftmp%2Fdocker-integration%2Fd98c044a4dea8.sock/v1.42/info": EOF: get swarm info
    --- FAIL: TestDockerSwarmSuite/TestSwarmLockUnlockCluster (66.10s)
```

--- a/hack/make/binary-proxy
+++ b/hack/make/binary-proxy
@@ -3,6 +3,8 @@
 set -e
 
 (
+	export CGO_ENABLED=0
+
 	GO_PACKAGE='github.com/docker/docker/cmd/docker-proxy'
 	BINARY_SHORT_NAME='docker-proxy'
 

